src/github/entities/allowed-organization.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { OrganizationStatus } from 'src/common/constants';
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity } from 'typeorm';
+
+@Entity()
+export class AllowedOrganization extends Base {
+  @Column({ type: 'varchar', length: 255, default: 'github' })
+  host: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true })
+  organizationName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminUserName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminEmail: string;
+
+  @Column('text', { array: true, default: [] })
+  activeUsers: string[];
+
+  @Column('text', { array: true, default: [] })
+  inactiveUsers: string[];
```

Comment:
PullProofAI

Consider initializing `activeUsers` and `inactiveUsers` arrays outside the class definition to improve performance.
```typescript
19:   @Column('text', { array: true, default: [] })
20:   activeUsers: string[] = [];
21: 
22:   @Column('text', { array: true, default: [] })
23:   inactiveUsers: string[] = [];
```

#### Issue 3: Redundant Default Value
28:
The default value for `status` is redundant due to the enum type.
```typescript
28:     enum: OrganizationStatus,
```

#### Issue 4: Unused Property
33:
The `allowAllUsers` property seems unused. Verify if it's necessary.
```typescript
33:   allowAllUsers: boolean;
```

These suggestions aim to enhance the code quality by addressing potential issues and improving performance where applicable.


<!-- This is an auto-generated comment by PullProofAI -->